### DISCUSSION TITLE
Problem maximising a client epoptes client window

### DISCUSSION THREAD

**Discussion by decenttp** [epoptes/epoptes#163] [Q&A]
Hi, 

I am having problem with clients where in I can view clients in the thumbnail view on epoptes server computer but unable to expand to full screen when clicked. What could be wrong?

**Comment by alkisg** [comment]
Hi, run `epoptes` on the server from a terminal,
and `/usr/sbin/epoptes-client` on a client from a terminal,
then reproduce the issue,
the paste both the terminal outputs here.
